cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/SelfHealingNotifier.java

```diff
@@ -233,8 +233,8 @@ public AnomalyNotificationResult onBrokerFailure(BrokerFailures brokerFailures)
       long delay = selfHealingTimeMs - nowMs;
       result = AnomalyNotificationResult.check(delay);
     } else {
-      // Reached auto fix threshold. Alert and fix if self healing is enabled.
-      boolean autoFixTriggered = _selfHealingEnabled.get(KafkaAnomalyType.BROKER_FAILURE);
+      // Reached auto fix threshold. Alert and fix if self healing is enabled and anomaly is fixable.
+      boolean autoFixTriggered = _selfHealingEnabled.get(KafkaAnomalyType.BROKER_FAILURE) && brokerFailures.fixable();
```

Comment:
This seems to subsume / hide the detail on non-fixable broker failure state. Can we make sure that this information is exposed in (1) alerts, (2) logs, and (3) anomaly detector state?